libs/core/src/components/pds-text/pds-text.scss

```diff
@@ -0,0 +1,187 @@
+:host {
+  display: block;
+}
+
+// TODO: Update all values to match new semantic tokens, if applicable
+
+/* stylelint-disable */
+h1 {
+  font-family: "GreetStandard";
+  font-size: 28px;
+  font-weight: 600;
+  letter-spacing: 0.26px;
+  line-height: 1.25;
+}
+
+h2 {
+  font-family: "GreetStandard";
+  font-size: 26px;
+  font-weight: 600;
+  letter-spacing: 0.24px;
+  line-height: 1.25;
+}
+
+h3 {
+  font-family: "GreetStandard";
+  font-size: 22px;
+  font-weight: 600;
+  letter-spacing: 0.22px;
+  line-height: 1.25;
+}
+
+h4 {
+  font-family: "GreetStandard";
+  font-size: 20px;
+  font-weight: 600;
+  letter-spacing: 0.20px;
+  line-height: 1.25;
+}
+
+h5 {
+  font-family: "GreetStandard";
+  font-size: 18px;
+  font-weight: 500;
+  letter-spacing: 0.18px;
+  line-height: 1.25;
+}
+
+h6 {
+  font-family: "GreetStandard";
+  font-size: 16px;
+  font-weight: 500;
+  letter-spacing: 0.16px;
+  line-height: 1.25;
+}
+
+code, em, p, pre, strong {
+  font-family: "Inter";
+  font-size: 14px;
+  font-weight: 400;
+  letter-spacing: -0.16px;
+  line-height: 1.425;
+}
+/* stylelint-enable */
+
+code, pre {
+  font-family: monospace;
+  letter-spacing: 0;
+  line-height: 1;
+}
+
+strong {
+  font-weight: bolder;
+}
+
+h1, h2, h3, h4, h5, h6, p, code, pre, em, strong {
+  color: var(--pine-color-text-default);
+  margin: 0;
+}
+
+:host([truncate]) > * {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+
+// Colors/Variants/Sentiment
+$type-colors: (
+  primary: var(--pine-color-white),
+  secondary: var(--pine-color-grey-800),
+  neutral: var(--pine-color-grey-900),
+  accent: var(--pine-color-purple-900),
+  danger: var(--pine-color-red-900),
+  info: var(--pine-color-blue-900),
+  success: var(--pine-color-green-900),
+  warning: var(--pine-color-yellow-900),
+);
+
+@mixin generate-type-colors($type-colors) {
```

Comment:
I understand the intention of using a mixin here; have we considered not using mixins in favor of explicit CSS declarations and less ambiguity?

Reply:
I'm not opposed to this. Is this a standard we'd like to adhere to going forward?

Reply:
Thanks for being open to feedback! Mixins are great for reusability, but explicit CSS can sometimes make the codebase clearer and easier to maintain. Let's get the message up to the team to see if this should become a standard.

Reply:
After discussing with the team in [Slack](https://kajabi.slack.com/archives/C03GKQJHCLX/p1733247972345769), I'm going to keep the mixins here for now as it's easy to see at a glance what values are being used and can be updated. However, this does come down to a strong case by case basis approach. For any and all future components, we'll be keeping a closer eye on mixin usage to ensure we don't fall into any weird patterns. If need be, we update this in the future if we decide otherwise.

Reply:
Great, thank you for the follow-up!